gapic/src/main/com/google/gapid/models/Profile.java

```diff
@@ -289,7 +251,27 @@ public boolean hasSlices() {
     }
 
     public List<PerfNode> getPerfNodes() {
-      return perfNodes;
+      return rootNode.children;
+    }
+
+    public PerfNode getPerfNode(Path.Commands link) {
```

Comment:
Local variable name "link" isn't obvious. Can you call it commandRange or forCommands to make it more clear what it's doing?

Reply:
Done.